.travis.yml

```diff
@@ -112,6 +113,9 @@ jobs:
     - stage: test
       script: ./ci/tests/validate-tests.sh
       name: "Validate Tests & Suites"
+    - stage: first
```

Comment:
I dont think this belongs here, or that it needs to be its own stage. It might be better to move this into the overlay itself, or simply move it to the validate stage. At a minimum, certainly `first` doesn't seem like an appropriate stage name.